src/ray/protobuf/gcs.proto

```diff
@@ -148,6 +154,14 @@ message ActorTableData {
   string class_name = 23;
   // Contains metadata about why the actor is dead.
   ActorDeathCause death_cause = 24;
+  // The max number of times tasks submitted on this actor should be retried
+  int64 max_task_retries = 25;
+  // Whether the actor supports out of order execution.
+  bool execute_out_of_order = 26;
+  // The max number of pending actor calls.
+  int32 max_pending_calls = 27;
+  // Quantities of the different resources required by this actor.
+  map<string, double> required_resources = 28;
```

Comment:
It seems that we don't have to put all these fields in the actor table. 
They are used for "GetNamedActorHandle". 
We can make GCS returns the actor table table + the task spec in the named actor query.

Reply:
yeah that's brilliant!